MAINTAINERS.yml

```diff
@@ -320,9 +319,9 @@ CMSIS-DSP integration:
         - "area: CMSIS-DSP"
 
 Common arch:
-    status: orphaned
-    collaborators:
+    maintainers:
         - dcpleung
+    collaborators:
         - nashif
```

Comment:
Should this be a separate commit/reviewed by the TSC since we are adding a maintainer here?

Also you removed `status:` here completely, which is not consistent with the rest (other maintained areas have `status: maintained`).

Reply:
yep, remove those for now, will come in a different PR